mobify/platform-scaffold
# Move app store to root and sdk

This PR moves some of the app state into it's own branch of the Redux store, and moves that new branch of the store into the SDK. 

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-21
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/697

## Changes
- Moved some app state into it's own branch of the store
- Moved that branch into the SDK

## How to test-drive this PR
- checkout the im-top-level-results branch for the SDK
- cd into the SDK folder
- run `npm link`
- run `npm run dev:build`
- cd back to the scaffold
- run `npm link progressive-web-sdk`
- run `npm run dev`
- preview the merlins site
- navigate through the site
- confirm everything still works correctly
